### PR TITLE
Fix queued emails

### PR DESCRIPTION
**Pull request by ryanong**
Still needs tests

### CODE REVIEW

**Comment by nickbarnwell** [comment]
Hey @ryanong, just wanted to restate how much I appreciate diving into a boring part of the codebase like email queuing and fixing it up. Definitely want to merge this in over the next couple of days, just wary of doing it right before a big event w.o tests. 

I'll write them for you next week if you don't get around to it or want to work on something else

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -21,6 +19,8 @@
 # If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.maintain_test_schema!
 
+load Rails.root + "db/seeds_test.rb"
+
```
Comment (mehulkar) [inline spec/spec_helper.rb]: why this change?

Comment (nickbarnwell) [inline spec/spec_helper.rb]: I don't think any of these seeds are necessary at this point. IIRC they were created right after I hacked in roles, but prior to factories.

**Comment by ankitshah811** [comment]
Is this relevant to #385? @nickbarnwell @ryanong

**Comment by mehulkar** [comment]
@ryanong I'm not sure of the original intention of this PR and I ruined everything with my force push :( Would you be able to cherry-pick the relevant commits and resubmit the PR so I can see the diffs?

**Comment by mehulkar** [comment]
closing due to inactivity.
